src/controller/sms/get_step_name.js

```diff
@@ -0,0 +1,18 @@
+const steps = require('./steps')
+const debug = require('debug')('sms') // eslint-disable-line
+
+/**
+ * getNextStepName - gets the current step name based on session data
+ *
+ * @param  {object} session session object from request
+ * @return {string} name of current step
+ */
+module.exports = function getStepName(session) {
+  const stepName = Object.keys(steps).reduce((memo, step) => {
+    if (memo) return memo
+    if (!session[step]) return step
```

Comment:
I _think_ we can store objects in the session. if so, the dispatcher could do `req.session.user = req.session.user || {}` and then do all our lookups in that object.

if not, we could prefix all our step keys with 'user_' when reading from/writing to the session. I'll try the object version and fall back to that if session has to be flat key/values.